tests/integration_test_dag.py

```diff
@@ -60,18 +63,22 @@ def add_constraint(table: Table):
     db = create_database(table.conn_id)
     if isinstance(db, SqliteDatabase):
         return "CREATE UNIQUE INDEX unique_index ON {{table}}(list,sell)"
+    # The only constraint that BigQuery supports is NOT NULL
+    # Hence, skipping the constraint for Bigquery.
+    if isinstance(db, BigqueryDatabase):
+        return "RETURN"
```

Comment:
@sunank200 what do you think if the `Database` classes exposed a method `setup_append`, which would:
* PostgresDatabase & Snowflake: create a constraint
* BQ: Do nothing
* SQLite: Create the index

This would allow us to not have to handle database-specific logic in this method, keeping it agnostic to database

Reply:
Sorry, @sunank200 , I meant `setup_merge` - they are needed for the merge operation, not append

Reply:
sure. I have named it to merge only.